packages/jellyfish-crypto/src/aes256.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Implementation reference: https://github.com/JamesMGreene/node-aes256
+ */
+import createHash from 'create-hash'
+import randomBytes from 'randombytes'
+import aes from 'browserify-aes'
+
+const CIPHER_ALGORITHM = 'aes-256-ctr'
+
+/**
+ * Encrypt a clear-text message using AES-256 plus a random Initialization Vector.
+ * @param {Buffer} key  A passphrase of any length to used to generate a symmetric session key.
+ * @param {Buffer} data  The clear-text message or buffer to be encrypted.
+ * @param {() => Buffer} initVector Initialization vector provider, then to create AES cipher, default using `crypto` or browserify `random-bytes` package
+ * @returns {Buffer}
+ */
+function encrypt (key: Buffer, data: Buffer, initVector?: () => Buffer): Buffer {
```

Comment:
```suggestion
function encrypt (key: Buffer, data: Buffer, rng?: (numOfBytes: number) => Buffer): Buffer {
```

Should be providing a rng, not providing an initVector

Reply:
You need to be extremely careful on how you implement nonce, don't give consumers of the library any chance of giving you wrong and unsafe vector.